src/components/StaleBundleWarning.js

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from 'react-query';
+import ky from 'ky';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Button, MessageBanner } from '@folio/stripes-components';
+import { useStripes } from '../StripesContext';
+
+const StaleBundleWarning = () => {
+  const stripes = useStripes();
+  const config = stripes?.config?.staleBundleWarning;
+  const refetchInterval = (config?.interval ?? 15) * 60 * 1000;
+  const [previous, setPrevious] = useState();
+  const [stale, setStale] = useState(false);
+  const query = useQuery({
+    queryKey: ['StaleBundleWarning'],
+    queryFn: async () => {
+      if (typeof config?.path !== 'string') return null;
+      try {
+        if (typeof config?.header === 'string') {
+          const res = await ky(config.path, { method: 'head' });
+          return res.headers.get(config.header);
+        } else {
+          return await ky.get(config.path).text();
+        }
+      } catch (e) {
+        console.warn(`'Error checking for new bundle ${e}`); // eslint-disable-line no-console
+        return null;
+      }
+    },
+    staleTime: refetchInterval,
+    refetchInterval,
+    enabled: !stale,
+  });
+  useEffect(() => {
+    if (!previous) {
+      setPrevious(query.data);
+      return;
+    }
+    if (previous !== query.data) setStale(true);
```

Comment:
Be like Perl: always put braces around conditionals. In fact, I can't believe we don't have a lint rule for this!

Reply:
I would also say add an extra line before that `if` too :)

Reply:
We do have a lint rule inherited from the airbnb ruleset we based our config on. It disallows multi-line if without braces but allows oneliners. I kinda like that, after all, do we put braces on our ternaries?

I did add some more whitespace though, it was a bit squished. Thanks!

Reply:
I feel like our lint rules let you down here, so that's definitely not on you. Filed [ESCONF-15](https://issues.folio.org/browse/ESCONF-15) to make braces mandatory so we can stop having debates like this. If our lint rules allow it, nobody should ever be allowed to comment on it :)

Reply:
I closed ESCONF-15; there is not widespread agreement around brace-formatting and besides, the _whole point_ of lint-rules is to avoid these kinds of debates on PRs. IOW, if lint didn't complain on this PR, then @zburke shouldn't be complaining on this PR. Srsly, some people 🙄 .